source/documentation/virtual_machine_management_guide/topics/Importing_an_OVA_file_from_a_host.adoc

```diff
@@ -17,7 +17,7 @@ The import process uses `virt-v2v`. Only virtual machines running operating syst
 +
 [NOTE]
 ====
-The location can be a local directory or a remote nfs mount, as long as it is not directly under `/root`. Ensure that it has sufficient space and is accessible to the qemu user (UID 36).
+The location can be a local directory or a remote NFS mount, as long as it is not in the`/root` directory or sub directories. Ensure that it has sufficient space.
```

Comment:
[IBM](https://www.ibm.com/docs/en/ibm-style?topic=elements-files-directories) recommends "subdirectory".

Reply:
the compound word was flagged in the editor as an error, will try subdirectory